launcher/launcher_windows.go

```diff
@@ -21,7 +21,7 @@ var (
 	createProcessW = kernel32.NewProc("CreateProcessW")
 )
 
-func runProcess(dir, command string) {
+func runProcess(dir, command, _entrypoint string) {
```

Comment:
Should this be named `entrypoint` instead?

Reply:
I don't mind renaming this to `entrypoint`, but we did deliberately add the underscore at the beginning to signal that the argument is currently unused for windows launchers. I don't feel strongly about it -- let me know if you'd like me to rename it, and I'll push a change.